src/igvfd/schemas/measurement_set.json

```diff
@@ -54,6 +54,20 @@
             "description": "Link to the protocol for conducting the assay on Protocols.io.",
             "type": "string",
             "pattern": "^https://www\\.protocols\\.io/(\\S*)$"
+        },
+        "construct_library": {
```

Comment:
I think this one has to be plural? array of relevant construct libraries?

Reply:
Makes sense, adjusted to be plural- we talked about restricting to 1 ConstructLibrary (similar to the logic for 1 SequencingLibrary, 1 Biosample per MeasurementSet) which is why maxItems is 1 (see below comment), but we would keep it an array in case anyone needed to submit multiple. I'm fine with removing maxItems as I don't think it would necessarily create confusion to have multiple